### ISSUE TITLE
Github discussions for community engagement

### ISSUE THREAD

**Issue by fliepeltje** [standard-for-public-code/standard-for-public-code#937]
It would be great to have a forum-like experience to engage with you guys on questions that do not fit very well in the github issue format; for instance it would be great to have a slimmed down version of the standard for scripts used in scientific research and it would be nice to have a longer discussion with input from various sources on what would be viable to set as a standard for such works.

**Comment by Ainali** [comment]
Hello @fliepeltje! For now, we have a [mailing list](https://lists.publiccode.net/mailman/postorius/lists/standard.lists.publiccode.net/) for this, which can be read in [a forum like style](https://lists.publiccode.net/hyperkitty/hyperkitty/list/standard@lists.publiccode.net/) . (It is mentioned in our [CONTRIBUTING file](https://github.com/publiccodenet/standard/blob/develop/CONTRIBUTING.md) but perhaps it should be made easier to find.) Would that be good enough for your purposes?

**Comment by fliepeltje** [comment]
I think in the broader sense, no :see_no_evil: 

There's a couple of reasons:
* The mailing list is not a very nice way to have Q&A style questions
* Since the standard is maintained on github, it is much easier to "find" the discussion if it were on github
* To start a new thread on the suggested space, I need to sign up or make an account. I guess you can start a thread through the mailing list, but I personally don't find that very intuitive and I know my colleagues are not fans of this either
* There's a bit of a barrier using the mail method as it can feel like I'm spamming people with my writing. I personally would prefer to have my ideas and feedback be available publicly, but not spammily :grimacing: 

A typical scenario for me would be that you might ask for support with complying to the standard through clarification of criteria or tips or suggestions and to have this discussion be visible. It is not nice to have it in a broad mailing group as there might be many emails going back and forth and that might encourage people to unsubscribe. It is not nice to do it in private as the knowledge and process are then not open and transparent. 

Ofcourse I would love to hear motivations and reasons for sticking with the current system (or avoiding the us of gh discussions) :)

**Comment by Ainali** [comment]
I don't think we have actively been avoiding the GitHub discussions (after all, [we use it in our Governance game repo](https://github.com/publiccodenet/governance-game/discussions)). It wasn't available as a feature when we started the Standard, and then we chose mailing lists as it is a tried and tested method, used in plenty of open source software communities. But as one can see, the traffic on the mailing list is really low, so perhaps we should try something new.

Usually, when it comes to trying to get a codebase into compliance, codebase stewards at the Foundation for Public Code are readily available for direct email or even video calls, which have lessened the demand for public places for these kinds of questions. It all becomes open in the end, since the evaluation itself is made open and any learnings are documented in the [implementation guide](https://publiccodenet.github.io/community-implementation-guide-standard/). That said, I do see the point of having some questions in the open. Let us think a bit about this.

**Comment by Ainali** [comment]
We have been discussing this and see some advantages with the GitHub discussions. We will just look into the technical possibilities if we want to migrate to another platform in the future before deciding.

**Comment by ericherman** [comment]
A quick glance at https://docs.github.com/en/graphql/guides/using-the-graphql-api-for-discussions looks promising, no obvious road-blocks.

It would be nice to see if a link can be sent to the mailing list when a new discussion is started, not essential.

As the Foundation uses Mattermost, we should look to see if we can get the GitHub plugin configured to notify on new discussion items.

**Comment by Ainali** [comment]
This will be discussed in [the next community call](https://write.publiccode.net/pads/Community-Call-Standard-for-Public-Code) on Thursday, 15.00 CEST.

**Comment by Ainali** [comment]
Here are the [notes from the community call](https://blog.publiccode.net/community%20call/2023/07/06/notes-from-community-call-6-july-2023.html).

As mentioned there, we would like to see a couple of examples of communities using GitHub discussions, both to establish that it is a good practice, and also to see how the functionality is being used in practice.

**Comment by clausmullie** [comment]
As we are currently using Github discussions, I will close this issue for now
